src/spec-node/featuresCLI/utils.ts

```diff
@@ -0,0 +1,118 @@
+export const staticProvisionParams = {
```

Comment:
I would love if we refactored the code a bit to make the `provision` and `exec` command easier to call internally.  I couldn't seem to find a good blend of "easy to use" without moving or copying a bunch of code :)